.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-and-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        run: ./gradlew build -x minniekenny # skip minniekenny -- run it separately in minnie-kenny.yaml
+      - name: Test
+        run: ./gradlew test -x minniekenny # skip minniekenny -- run it separately in minnie-kenny.yaml
```

Comment:
Instead of skipping minniekenny twice, another option is to move the contents of minnie-kenny into this yaml.

Reply:
What would moving minnie-kenny into this yaml do? Doesn't it need to be run locally on a dev's machine to verify that they have the right things installed?

Reply:
just to clarify, there's a minnie-kenny.yaml that sets up minnie-kenny to run on github actions. The intention there is so that if a user does not have minnie-kenny set up locally, the commits will still get scanned at the PR level. We could combine both yamls. I opted to separate them but I'm not super tied to it.

Reply:
Per offline discussion, I don't think there's a ton of value in running minnie-kenny once secrets have already leaked on a push to github. I'd say don't bother so we have less things to maintain and break and rely on the typical scans for finding commits that leak secrets.
Separating them does seem good if we go ahead with a minnie-kennie on github action.

Reply:
Removed minnie-kenny github action for these reasons as stated above.